app/components/review_action_component.rb

```diff
@@ -0,0 +1,45 @@
+class ReviewActionComponent < ViewComponent::Base
+  def initialize(review_action:, application:)
+    @application = application
+    @action = review_action
+
+    super
+  end
+
+  attr_reader :application, :action
+
+  def call
+    govuk_button_to(button_text, target, method:, warning:)
+  end
+
+  private
+
+  def button_text
+    I18n.t(action, scope: 'calls_to_action')
+  end
+
+  def target
+    case action
+    when :complete
+      complete_crime_application_path(application)
+    when :send_back
+      new_crime_application_return_path(application)
+    when :mark_as_ready
+      ready_crime_application_path(application)
+    end
+  end
+
+  def method
+    return :get if confirmed?
+
+    :put
+  end
+
+  def confirmed?
```

Comment:
Not sure I understand why this method is named confirmed

Reply:
It's used to indicate if the action is confirmed ("Are you sure?"). I'll try to re-word.

Reply:
I have removed the method altogether as it was misleading.